lib/util/sudo_debug.c

```diff
@@ -536,7 +536,7 @@ sudo_debug_exit_id_t_v1(const char *func, const char *file, int line,
 	"<- %s @ %s:%d := %lld", func, file, line, (long long)ret);
 #else
     sudo_debug_printf2(NULL, NULL, 0, subsys | SUDO_DEBUG_TRACE,
-	"<- %s @ %s:%d := %d", func, file, line, (int)ret);
+	"<- %s @ %s:%d := %ld", func, file, line, (long)ret);
```

Comment:
I don't see a need for this change.  If `id_t` is 32-bits, printing it as an int should be fine.